src/landing-page/create-modal.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import {
+	// eslint-disable-next-line
+	__experimentalHStack as HStack,
+	// eslint-disable-next-line
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line
+	__experimentalSpacer as Spacer,
+	// eslint-disable-next-line
+	__experimentalText as Text,
+	Modal,
+	Button,
+	TextControl,
+	TextareaControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	createBlankTheme,
+	createClonedTheme,
+	createChildTheme,
+} from '../resolvers';
+
+export const CreateThemeModal = ( { onRequestClose, creationType } ) => {
+	const [ errorMessage, setErrorMessage ] = useState( null );
+
+	const [ theme, setTheme ] = useState( {
+		name: '',
+		description: '',
+		author: '',
+	} );
+
+	const createBlockTheme = async () => {
+		let constructionFunction = null;
+		switch ( creationType ) {
+			case 'blank':
+				constructionFunction = createBlankTheme;
+				break;
+			case 'clone':
+				constructionFunction = createClonedTheme;
+				break;
+			case 'child':
+				constructionFunction = createChildTheme;
+				break;
+		}
+
+		if ( ! constructionFunction ) {
+			return;
+		}
+		constructionFunction( theme )
+			.then( () => {
+				// eslint-disable-next-line
```

Comment:
Specify which rules are being disabled, so that it's easy to tell why at a glance.

```suggestion
				// eslint-disable-next-line no-undef, no-alert
```